.circleci/real_config.yml

```diff
@@ -391,18 +388,17 @@ commands:
 
   install-wheel:
     parameters:
-      package-name:
-        type: string
       package-location:
         type: string
     steps:
       - run:
-          name: Install <<parameters.package-name>>
```

Comment:
this previously required `pipeline.parameters.det-version`, so idk why were were parameterizing on package name

Reply:
@dannysauer

Reply:
It looks like this could install any package the same way instead of only installing Determined. 🤷‍♂️ If it's only used for one package, then I agree that the parameter is unecessary.

(FWIW, the blame probably shows that I wrote all this stuff because I renamed the config file when switching to dynamic config. :))

Reply:
ohh okay! thanks for looking